packages/types/src/connection/manager.ts

```diff
@@ -0,0 +1,30 @@
+import { RequestContext } from "../transfer";
+import { ConnectConfiguration } from "./config";
+
+export interface ConnectionManagerConfiguration {
+  /**
+   * Maximum number of allowed concurrent requests per connection.
+   */
+  maxConcurrency?: number;
+
+  disableConcurrency?: boolean;
```

Comment:
this is equivalent to maxConcurrency = 1 and overrides that field?

Reply:
Yes, the field will be deprecated and handle to maintain backwards compatibility

Reply:
No, `disableConcurrency` controls disabling/enabling concurrency. 

For http2, a [`session`](https://nodejs.org/api/http2.html#http2_class_clienthttp2session) contains multiple [stream](https://nodejs.org/api/http2.html#class-http2stream) instances.

The current `NodeHttp2Handler` implementation creates a new http2 `session` per url request when `disableConcurrency` is set to `true`. i.e. not reusing existing session and not utilizing concurrent streams

`maxConcurrency` dictates the number of concurrent streams per session. By default, a http2 session allows a maximum of 2 ^ 31 - 1 stream instances.
If `disableConcurrency` is set to `true`, then `maxConcurrency` becomes meaningless.